src/backend/dx12/src/lib.rs

```diff
@@ -542,6 +545,15 @@ impl Device {
             device::CommandSignature::Dispatch,
         );
 
+        let id = unsafe {
+            // LUID is a 64 bit number
+            let luid = device.GetAdapterLuid();
```

Comment:
Unsure about the choice of ID as this is adapter specific and not unique to each device (logical devices would have the same). Currently we only support one logical device per adapter but we might lift that again.